### DISCUSSION TITLE
How to change Rustdesk Server key on Ubuntu

### DISCUSSION THREAD

**Discussion by henriquelucas** [rustdesk/rustdesk-server#351] [General]
I did a fresh installation of Rustdesk on my Ubuntu Server.

I had a key:
Focn1GbvFJBfOWrHduUmrEifjVNtvVTFtc6QmQOt7TU=

Would it be possible to put this key back in so I wouldn't have to reconfigure all the devices again? And yes, how do I do this in Ubuntu, which command?

**Comment by bigretromike** [comment]
You have the shorter key, so its public key. the way this kind of cryptography works is that with Public Key you are UNABLE to generate PRIVATE Key.

**Comment by bato3** [comment]
During installation keys are generated: `id_ed25519  id_ed25519.pub`. You can use any valid pair of keys (type: ed25519).

In my case they are stored in Rustdesk directory: `/opt/rustdesk/`

You can check where are binaries by `ps -aux|grep hb`
